shipyard.gemspec

```diff
@@ -12,16 +12,16 @@ Gem::Specification.new do |spec|
   spec.license       = 'MIT'
 
   # Rails Dependencies
-  spec.add_runtime_dependency 'actionview', '~> 5.0'
-  spec.add_runtime_dependency 'sprockets-es6', '~> 0.9.2'
+  spec.add_runtime_dependency 'actionview', '~> 5.2.0'
+  spec.add_runtime_dependency 'sprockets', '~> 4.0.0.beta8'
```

Comment:
@jsouth Here's what fixed the issue. The new `sprockets` also handles `es6` files as well now, which is basically the new JavaScript format.

Reply:
Sweet 👍 

Great work, and thank you for fixing this!